### ISSUE TITLE
WSL Sequencer (FPC): Failure running fpcupdeluxe

### ISSUE THREAD

**Issue by mr-highball** [LongDirtyAnimAlf/fpcupdeluxe#285]
Hello! I've attempted to install lazarus/fpc on [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) but am running into install issues that I'm unsure of.
I've followed some advice on this forum [post](https://forum.lazarus.freepascal.org/index.php/topic,34645.1590.html) such installing these packages first,
`sudo apt-get install gdb libx11-dev libgtk2.0-dev libgdk-pixbuf2.0-dev libcairo2-dev libpango1.0-dev`

Here are the details of my setup/fpcupdeluxe used (latest at the time but have tried earlier)
OS: Ubuntu 20.04 LTS
Ver: Release v1.8.0d of fpcupdeluxe - fpcupdeluxe-x86_64-linux

Here is the full error:
```
/usr/bin/make packages_smart FPC=/home/XXX/fpcupdeluxe/stable/fpcsrc/compiler/ppcx64 FPCFPMAKE=/home/XXX/fpcupdeluxe/stable/fpcsrc/compiler/ppcx64 RELEASE=1 'OPT=-Sg -vw-n-h-l-d-u-t-p-c-'
/usr/bin/make -C packages smart
/usr/bin/make -C fpmkunit bootstrap
/home/XXX/fpcupdeluxe/stable/fpcsrc/compiler/ppcx64 src/fpmkunit.pp  -Fu/home/XXX/fpcupdeluxe/stable/fpcsrc/rtl/units/x86_64-linux -FUunits_bs/x86_64-linux -Fu../paszlib/src -Fu../hash/src -Fi../paszlib/src -Cg  -Fi../fcl-process/src/unix -Fu../fcl-process/src -Fi../fcl-process/src/linux -Fi../fcl-process/src/dummy -Fu../libtar/src -Sg -vw-n-h-l-d-u-t-p-c-
/home/XXX/fpcupdeluxe/stable/fpcsrc/compiler/ppcx64 fpmake.pp  -Fu/home/XXX/fpcupdeluxe/stable/fpcsrc/packages/fpmkunit/units_bs/x86_64-linux -Fu/home/XXX/fpcupdeluxe/stable/fpcsrc/rtl/units/x86_64-linux  -Sg -vw-n-h-l-d-u-t-p-c-
./fpmake compile "-T 8" --localunitdir=.. --os=linux --cpu=x86_64 -o -Ur -o -Xs -o -O2 -o -n -o -Cg -o -Sg -o -vw-n-h-l-d-u-t-p-c- -o -dx86_64 -o -dRELEASE --compiler=/home/XXX/fpcupdeluxe/stable/fpcsrc/compiler/ppcx64 -bu -o -XX -o -CX
make[1]: *** [Makefile:2681: packages_smart] Error 2
make: *** [Makefile:2836: build-stamp.x86_64-linux] Error 2
Invalid command-line argument at position 2: -T 8
Usage: /home/XXX/fpcupdeluxe/stable/fpcsrc/packages/fpmake command [options]
Where command is one of the following:
 compile      Compile all units in the package(s).
 build        Build all units in the package(s).
 install      Install all units in the package(s).
 buildinstall Build and install all units in the package(s).
 uninstall    Uninstall the package(s).
 clean        Clean (remove) all generated files in the package(s) for current CPU-OS target.
 distclean    Clean (remove) all generated files in the package(s) for all targets.
 archive      Create archive (zip) with all units in the package(s).
 manifest     Create a manifest suitable for import in repository.
 zipinstall   Install all units in the package(s) into an archive.
 pkglist      Create list of all packages suitable for FPC installer.
Where options is one or more of the following:
 -h --help             This message.
 -l --list-commands    list commands instead of actually executing them.
 -n --nofpccfg         Compiler will not use fpc.cfg
 -v --verbose          Be verbose when working.
 -d --debug            Add debug information when working.
 -I --interactive      Allow to interact with child processes
 -e --useenv           Use environment to pass options to compiler.
 -ie --installexamples  Install the example-sources.
 -bu --buildunit        Compile package in Build-unit mode.
 -scp --skipcrossprograms Skip programs when cross-compiling/installing
 -io --ignoreinvalidoption Ignore further invalid options.
 -C --cpu=Value            Compile for indicated CPU.
 -O --os=Value             Compile for indicated OS
 -t --target=Value         Compile for indicated target
 -P --prefix=Value         Use indicated prefix directory for all commands.
 -B --baseinstalldir=Value Use indicated directory as base install dir.
 -BI --bininstalldir=Value  Use indicated directory as base install dir.
 -LI --libinstalldir=Value  Use indicated directory as base install dir.
 -UL --localunitdir=Value   Use indicated directory as local (user) unit dir.
 -UG --globalunitdir=Value  Use indicated directory as global unit dir.
 -sp --searchpath=Value     Add search directory for packages.
 -U --unitinstalldir=Value Use indicated directory to install units into.
 -r --compiler=Value       Use indicated binary as compiler
 -f --config=Value         Use indicated config file when compiling.
 -o --options=Value        Pass extra options to the compiler.
 -df --doc-folder=Value     Use indicated directory as fpdoc output folder.
 -fsp --fpunitsrcpath=Value  Sourcepath to replace in fpunits.cfg on installation.
 -zp --zipprefix=Value      Use indicated prefix for generated archives.
 -T --threads=Value        Enable the indicated amount of worker threads.
    --data2inc=Value       Use indicated data2inc executable.
    --genfpmkunit=Value    Regenerate the fpmkunitsrc.inc file (fppkg).
    --CompilerTarget=Value Target CPU for the IDE's compiler
    --NoGDB=Value          If value=1 or 'Y', no GDB support
    --NoGDBMI=Value        If value=1 or 'Y', explicitly disable GDB/MI option
    --GDBMI=Value          If value=1 or 'Y', builds IDE with GDB/MI support (no need for LibGDB)
    --NoIDE=Value          If value=1 or 'Y', the IDE will be skipped
    --IDE=Value            If value=1 or 'Y', the IDE will be build for each target
To add a package-variant to all packages:
 +[variantname]+=[variant1],<variant2>,...
To add a package-variant to all packages which other packages will inherit:
 +[variantname]*=[variant1],<variant2>,...
To add specific options for one package-variant:
 --options_[variantname]_[variant1]=Value

fpcupdeluxe: ERROR: FPCNativeInstaller (BuildModuleCustom: FPC): Error running /usr/bin/make for FPC failed with exit code 512
. Details: 


ERROR: Fpcupdeluxe fatal error !
Sequencer (FPC): Failure running fpcupdeluxe: error executing sequence FPC
Sequencer (Default): Failure running fpcupdeluxe: error executing sequence Default
```

Any suggestions would be appreciated, thanks!

**Comment by LongDirtyAnimAlf** [comment]
Sorry about this.
However.
This error is/was caused by a bugger in 1.8.0c and should have been solved in 1.8.0d !
Are you sure you are using 1.8.0d ?

**Comment by mr-highball** [comment]
I apologize, I must've downloaded `c` just before you released `d` since I did this yesterday. I've confirmed that the newest release correctly installs lazarus/fpc. Again, I appreciate your quick response and love fpcupdeluxe (I try to tell everyone about it for new installs)
I'm going to close this issue now
